src/Stuart/Client.php

```diff
@@ -31,6 +33,24 @@ public function validateJob(Job $job)
         }
     }
 
+    /**
+     * @param $city
+     * @param $type pickup or dropoff
+     * @param \DateTime $dateTime
+     *
+     * @return SchedulingSlots|object
+     */
+    public function getSchedulingSlots($city, $type, \DateTime $dateTime)
```

Comment:
I would create an Enum to handle the City, it could be use as following: `\Stuart\City::PARIS`.
the method could be called as follow: 
```php
$date = \DateTime::createFromFormat('Y-m-d\TH:i:s.uO', '2017-07-20T08:45:00.000+01:00');
$client->getSchedulingSlotsAtDropoff(\Stuart\City::PARIS, $date);
```